playbooks/roles/capsule/tasks/devel_install.yml

```diff
@@ -0,0 +1,19 @@
+- name: 'start rails server'
+  command: /home/vagrant/.rvm/bin/rvm ruby-2.2.4 do rails s
```

Comment:
@ehelms The only issue I see with that is if the developer has a server already running and uses this automation, it would kill their running server which may not be what they want. I agree if it isn't running and we start it in this playbook, we should stop it as well, if there is a way to just do that.